src/cryptenroll/cryptenroll-tpm2.c

```diff
@@ -142,23 +142,27 @@ int enroll_tpm2(struct crypt_device *cd,
         _cleanup_(erase_and_freep) void *secret = NULL;
         _cleanup_(json_variant_unrefp) JsonVariant *v = NULL, *signature_json = NULL;
         _cleanup_(erase_and_freep) char *base64_encoded = NULL;
+        _cleanup_(erase_and_freep) char *secret_and_salt = NULL;
```

Comment:
fold above

Reply:
Thanks for the great feedback.
By fold do you mean "put the two declarations into one line" or do you mean "reuse the variable above"?

Reply:
two decl on one line